src/components/navigation/Topbar.tsx

```diff
@@ -40,7 +40,7 @@ const useStyles = makeStyles((theme) => ({
     margin: 'auto',
     padding: theme.spacing(0, 1),
     display: 'grid',
-    gridTemplateColumns: '170px 1fr 170px',
+    gridTemplateColumns: '170px 1fr 172px', //*Her har jeg endret med 2px for å få plass til alle 3 ikoner til høyre
```

Comment:
Hvis du legger til 2px på begge sider så vil vel menyen fremdeles være sentrert. Du trenger heller ikke legge inn kommentar i koden på slike endringer. Hvis det er nyttig for reviewer å vite grunn til en endring (slik som her), så kan du for eks heller legge til en kommentar som dette 🚀 

```suggestion
    gridTemplateColumns: '172px 1fr 172px',
```